src/libpmem/aarch64/init.c

```diff
@@ -51,7 +51,8 @@ memmove_nodrain_libc(void *pmemdest, const void *src, size_t len,
 	(void) flags;
 
 	memmove(pmemdest, src, len);
-	pmem_flush(pmemdest, len);
+	if (!(flags & PMEM_MEM_NOFLUSH))
```

Comment:
maybe we could have a global macro:
`pmem_flush_flags(pmemdest, len, flags)` ?
Would allow you to move the flags checking to a single function.

Reply:
Done.